frontend/src/libs/components/select/select.tsx

```diff
@@ -22,19 +23,23 @@ type Properties<T extends FieldValues> = {
   name: FieldPath<T>;
   options: SelectOption<string | number>[];
   placeholder?: string;
+  label?: string;
```

Comment:
```suggestion
  label: string;
```
label should be always